### PR TITLE
feat: removing unnecessary route

### PR DESCRIPTION
**Pull request by carlosctu**
### Descrição

- [x] Removi rota do App.js

### Links

[Trello](https://trello.com/c/5usTK7d5)

### Screenshots

image

### CODE REVIEW

**Review by matheusnamorim — COMMENTED** [review]
uau

File: package.json [hunk 1]
```diff
@@ -10,6 +10,7 @@
     "@testing-library/react": "^11.2.7",
     "@testing-library/user-event": "^12.8.3",
     "axios": "^0.21.1",
+    "babel-jest": "^29.3.1",
     "date-fns": "^2.23.0",
     "dayjs": "^1.10.6",
     "file-saver": "^2.0.5",
```
Comment (carlosctu) [inline package.json:16]: alkdsjlkadalkjda

File: src/App.js [hunk 2]
```diff
@@ -46,7 +46,6 @@ export default function App() {
                 <Route path="hotel" element={<Hotel />} />
                 <Route path="activities" element={<Activities />} />
                 <Route path="certificate" element={<Certificate />} />
-                <Route index path="*" element={<Navigate to="/dashboard/subscription" />} />
```
Comment (carlosctu) [inline src/App.js:49]: asdasd

File: package.json [hunk 3]
```diff
@@ -10,6 +10,7 @@
     "@testing-library/react": "^11.2.7",
     "@testing-library/user-event": "^12.8.3",
     "axios": "^0.21.1",
+    "babel-jest": "^29.3.1",
     "date-fns": "^2.23.0",
```
Comment (carlosctu) [inline package.json:14]: adasdasdasdas

File: package.json (hunk 1 above)
Comment (matheusnamorim) [inline package.json:16]: entendi

**Pull request closed without merging**